### PR TITLE
Adds ANZAC day

### PR DESCRIPTION
**Pull request by optimumtact**
Biscuit moment

## Changelog
:cl: oranges
add: Added ANZAC day
/:cl:

### CODE REVIEW

**Comment by theboy6545** [comment]
Australia and new Zealand don't exist

File: code/modules/holiday/holidays.dm [hunk 1]
```diff
@@ -281,6 +281,14 @@
 /datum/holiday/friendship/greet()
 	return "Have a magical [name]!"
 
+/datum/holiday/anz
+	name = "ANZAC Day"
+	begin_day = 25
+	begin_month = APRIL
+
+/datum/holiday/anz/getStationPrefix()
+	return pick("Australian","New Zealand","Poppy", "Southern Cross")
```
Comment (PowerfulBacon) [inline code/modules/holiday/holidays.dm]:
```suggestion
	return pick("Australian", "New Zealand", "Poppy", "Southern Cross")
```

**Comment by qwertyquerty** [comment]
image

**Review by PowerfulBacon — APPROVED** [review]

**Comment by optimumtact** [comment]
You should probably fix this
`PHP Fatal error:  Array and string offset access syntax with curly braces is no longer supported in ./tools/WebhookProcessor/github_webhook_processor.php on line 968`

**Comment by PowerfulBacon** [comment]
Yea its being sorted probably

**Comment by utilbird** [comment]
why did oranges PR this downstream and then upstream

**Comment by optimumtact** [comment]
priorities

**Pull request merged**